app/models/claim.rb

```diff
@@ -116,6 +123,22 @@ def update_disbursement_positions!
 
   private
 
+  def sorted_work_item_ids
+    @sorted_work_item_ids ||= work_items.sort_by do |workitem|
+      [
+        workitem.completed_on || 100.years.ago,
```

Comment:
so this won't do quite what you think... as `100.years.ago` returns a time value and it will be different for each row, hense records without a date will be arbitrarily sorted instead of following the worktype, created_at pattern... 

thsi also means the order may change on page refresh..
use this instead:
```
workitem.completed_on || Time.new(2000, 1, 1),
```
might also be worth checking if workj_type needs a default 
```
workitem.work_type&.downcase || '',
```

Reply:
ooh, good spot, albeit I think i checked this locally and it puts the "incompletes" at the top, by default, which is what we kind of want.

Reply:
done.